src/posts/flypados3.md

```diff
@@ -0,0 +1,173 @@
+
+There are other goodies for our pilots to dive into as well, including updated Navigraph support, a new and improved pushback feature with onboard map, lighting presets, and in-flight checklists!
+
+## New Features
+
+### A Refreshing Unified User Experience
+
+At FlyByWire, we believe in well-designed and integrated user interfaces that bring a great quality of life experience to our users. Since the introduction of our flyPad to MSFS, we've done away with copying real world operating systems - instead building a cohesive UI that easily integrates useful features at the tip of your cursor. Starting up the EFB requires a simple click on the screen, where you're greeted with our unique dashboard. From here, you can seamlessly download your simBrief OFP to display all the information required for 
+your planned flight. Everything that's important for your flight is on the new dashboard:
+
+- Shortcuts to take you to where you want to go.
+- Flight status and general information including your planned route.
+- METAR information - customizable with your choice of ICAO (displayed raw or with simple symbology).
+- Active and upcoming checklists.
+- Pinned Navigraph charts.
+- Active failure management.
+
+This user experience extends throughout our flyPadOS, enabling our pilots to quickly peruse their OFP, access calculations / charts, and integrate easily with online networks such as VATSIM or IVAO.
+
+As you navigate throughout the EFB accessing different features and interacting with them, flyPadOS 3 now remembers the state you left it in. It remembers the pages or tabs with the associated data you have entered, so you can safely switch to a new screen without losing any data.
+
+![Dashboard](/img/notam-images/flypados3/pinned-charts-widget.png)
+![Weather Widget](/img/notam-images/flypados3/weather-widget-input.png)
+![Fuel Dispatch](/img/notam-images/flypados3/flypad-dispatch-fuel.png)
+![OFP Dispatch](/img/notam-images/flypados3/flypad-dispatch-ofp.png)
+
+### Support for Over 30 Different Languages
+
+We are extremely happy to offer support for over 30+ languages onboard our EFB. A first in flight simulation, you can now access all the features mentioned above and interact with them in a language of your choice! This would not have been possible without the help of over 200 contributors, and those who have offered to review every corner of the EFB for accuracy. A huge benefit of an open source project is our unique position to have our community assist in endeavors such as this. We cannot stress enough how thankful we are.
+
+[Pictures Here]
+
+### Integrated Pushback
+
+Our new pushback model is smoother with fine control for speed and direction. You can easily utilize your keyboard or various controller axis to steer the tug as you pushback. 
```

Comment:
```suggestion
Our new pushback model is smoother with fine control for speed and direction. You can easily utilize your controller axis to steer the tug as you pushback. 
```

No keyboard

Reply:
Fixed in a PR submitted into Val's fork.